src/libs/modals/modals/ModalConfirmStrategy/ModalConfirmStrategy.tsx

```diff
@@ -22,8 +23,13 @@ export const ModalConfirmStrategy: ModalFC<ModalConfirmStrategyData> = ({
   data: { strategy, type },
 }) => {
   const { closeModal } = useModal();
-  const { pauseStrategy } = useUpdateStrategy();
-  const { deleteStrategy } = useDeleteStrategy();
+  const { pauseStrategy, isProcessing, setIsProcessing, updateMutation } =
+    useUpdateStrategy();
+
+  const { deleteStrategy, deleteMutation } = useDeleteStrategy();
+  const isAwaiting =
+    type === 'delete' ? deleteMutation.isLoading : updateMutation.isLoading;
```

Comment:
yeah this is really not ideal. Lets open a follow up issue or better even to add to the deep link issue that we need to decouple these edit features.

Reply:
Added this issue to deeplink issue